patches/content-browser-renderer_host-render_process_host_impl.cc.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/content/browser/renderer_host/render_process_host_impl.cc b/content/browser/renderer_host/render_process_host_impl.cc
+index 352f6d2ecd0acb2937f67e519df02f930e3bb4bb..3db238ddf1025d74a4fc1f715b86f4c248593e10 100644
+--- a/content/browser/renderer_host/render_process_host_impl.cc
++++ b/content/browser/renderer_host/render_process_host_impl.cc
+@@ -3399,12 +3399,15 @@ void RenderProcessHostImpl::AppendRendererCommandLine(
+     command_line->AppendSwitch(switches::kNoZygote);
+   }
+ 
+-  if (IsJitDisabled()) {
+-    command_line->AppendSwitchASCII(blink::switches::kJavaScriptFlags,
+-                                    "--jitless");
+-  } else if (AreV8OptimizationsDisabled()) {
+-    command_line->AppendSwitchASCII(blink::switches::kJavaScriptFlags,
+-                                    "--disable-optimizing-compilers");
++  if (IsJitDisabled() || AreV8OptimizationsDisabled()) {
```

Comment:
Rather than change this logic, command line options can be adjusted based on a brave-internal flag in `BraveMainDelegate::AppendCommandLineOptions`

Reply:
The actual feature flag would go in brave_browser_features probably